src/coord/src/catalog.rs

```diff
@@ -1733,6 +1747,10 @@ impl sql::catalog::Catalog for ConnCatalog<'_> {
     fn is_materialized(&self, id: GlobalId) -> bool {
         !self.catalog.indexes[&id].is_empty()
     }
+
```

Comment:
This was moved to appease clippy!

Reply:
Weird, do you remember what lint? Maybe we should disable it. Seems annoying.

Reply:
Oops - the warning was actually from CLion! `"Different impl member order from the trait"`